examples/pylint/W0125_using_constant_test.py

```diff
@@ -0,0 +1,13 @@
+"""pylint: using constant test
+
+"""
+
+
+def pos():
+    """
+    @rtype: bool
+    """
+    if 0:
```

Comment:
Can you do `if 0 > 5` and get the error? I don't want to see `0` used as a condition, but would also be okay with `if True`